### ISSUE TITLE
[COGNITION]: Breadcrumb - Links should announce as list with counts to VoiceOver

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#10464]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

Our [Breadcrumb component](https://design.va.gov/components/breadcrumbs) has an unnecessary `aria-live` attribute, and the list items are not announcing correctly to VoiceOver. I'd like to update these two things with the snippets highlighted below.

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

- [ ] List of links is counted correctly. Should say "List, 3 items" when navigating to the UL in VoiceOver
- [ ] Individual links should say "Link name, 1 of 3" in VoiceOver

## Environment

* MacOS Safari
* VoiceOver
* https://design.va.gov/components/breadcrumbs

## Possible Fixes (optional)

```diff
! Breadcrumbs.jsx#L37

return (
  <nav
    aria-label={ariaLabel}
-   aria-live="polite"
    className={classnames({
      'va-nav-breadcrumbs': true,
      'va-nav-breadcrumbs--mobile': !!mobileFirstProp,
      [className]: !!className,
    })}
    data-mobile-first={mobileFirstProp}
    id={breadcrumbId}
  >
    <ul
      className="row va-nav-breadcrumbs-list columns"
      id={breadcrumbListId}
+     role="list"
    >
      {this.renderBreadcrumbLinks()}
    </ul>
  </nav>
);
```

## WCAG or Vendor Guidance (optional)

* [Name, Role, Value: Understanding SC 4.1.2](https://www.w3.org/TR/UNDERSTANDING-WCAG20/ensure-compat-rsv.html)

**Comment by 1Copenut** [comment]
Issue moved to [department-of-veterans-affairs/vets-design-system-documentation #214](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/214) via [**ZenHub**](https://www.zenhub.com/)
